### DISCUSSION TITLE
Any plans for Linux OS?

### DISCUSSION THREAD

**Discussion by WhiteH4tDude** [srounet/Pymem#122] [Ideas]
Pymem supports only Windows for now. Is there any plans to make it for Linux? 

For scanning, writing and reading the process memory, I'm using [PyMemoryEditor](https://github.com/JeanExtreme002/PyMemoryEditor) library, that supports Windows and Linux. I would like more features for Linux.

**Comment by StarrFox** [comment]
I think this would be nice to have, I don't know much about linux's api though

**Comment by srounet** [comment]
Reversing for Linux Operating System is totally another project, processes do not work the same nor memory allocation and I'm not even taking about syscalls.

It would be great to have a unique library that could be used as a single interface to operate with processes but as far as I know it is not really achievable, as Windows, Linux and OSX are very different.
